collectors/health.go

```diff
@@ -27,16 +27,43 @@ import (
 // It surfaces changes in the ceph parameters unlike data usage that ClusterUsageCollector
 // does.
 type ClusterHealthCollector struct {
+	// conn holds connection to the Ceph cluster
 	conn Conn
 
-	DegradedPGs   prometheus.Gauge
-	UncleanPGs    prometheus.Gauge
+	// DegradedPGs shows the no. of PGs that have some of the replicas
```

Comment:
Seems to be a bit of inconsistency in these comments between "no." and "count".  Mind picking one and using it throughout?  Apologies for the nitpick.